msteams-platform/messaging-extensions/how-to/search-commands/define-search-command.md

```diff
@@ -9,65 +9,79 @@ ms.author: anclear
 
-The prerequisite to create a search command is that you must already create a messaging extension. For information on how to create a messaging extension, see [create a messaging extension](~/messaging-extensions/how-to/create-messaging-extension.md).
+The prerequisite to create a search command is that you must already have created a messaging extension. For information on how to create a messaging extension, see [create a messaging extension](~/messaging-extensions/how-to/create-messaging-extension.md).
 
 **To create a search command**
 
-1. From the Microsoft Teams client, open **App Studio**, and select the **Manifest editor** tab.
-1. If you already created an app package in the **App Studio**, choose it from the list. If you have not created an app package, import an existing one.
-1. After importing an app package, select **Messaging extensions** under **Capabilities**.
-1. Select **Add** in the **Command** section in the messaging extensions page.
-1. Choose **Allow users to query your service for information and insert that into a message**.
+1. Open **App Studio** from the Microsoft Teams client, and select the **Manifest Editor** tab.
+1.  If you already created your app package in **App Studio**, choose it from the list. If you have not created an app package, import an existing one.
+1. After importing app package, select **Messaging extensions** under **Capabilities**. You get a pop-up window to set up the messaging extension.
+1. Select **Set up** in the window to include the messaging extension in your app experience. The following image displays the messaging extension set up page: 
+
+    ![messaging extension set up](../../../assets/images/messaging-extension/messaging-extension-set-up.png)
+
+1. To create the messaging extension, you need a Microsoft registered bot. You can either use an existing bot or create a new bot. Select **Create new bot** option, give a name for the new bot, and select **Create**. The following image displays bot creation for messaging extension:
+
+    ![create bot for messaging extension](../../../assets/images/messaging-extension/create-bot-for-messaging-extension.png)
+
+1. Select **Add** in the **Command** section of the messaging extension page.
+1. Choose **Allow users to query your service for information and insert that into a message**. The following image displays the search command parameter selection:
+
+    ![search command parameter selection](../../../assets/images/messaging-extension/search-command-parameter-selection.png)
+
 1. Add a **Command Id** and a **Title**.
-1. Select the location from where your search command must be triggered. Selecting **message** does not currently alter the behavior of your search command.
+1. Select the location from where your search command must be triggered. Selecting **message** does not currently alter the behavior of your search command. The following image displays the search command invoke locatio:
```

Comment:
This createToDo sounds like an action command rather than search. Could you please use this screenshot or use another one if you have?

![image](https://user-images.githubusercontent.com/31851992/112617510-10c57700-8e4b-11eb-8002-d836f297f791.png)